spec/beaker_test_helpers.rb

```diff
@@ -0,0 +1,17 @@
+# These are specifically to mock Beaker methods necessary for testing
+# that will be available during runtime because this is never run separate
+# from Beaker itself.
+#
+# Including Beaker as a dependency would not work as a solution to this issue,
+# since that would make a cycle in the dependency graph, at least until
+# Beaker 3.0 happens and this is no longer a dependency of Beaker's.
+module BeakerTestHelpers
```

Comment:
@tvpartytonight is it ambiguous? A lot of times, I think that having names repeated when they're already available is just useless redundancy that should be eliminated. In this case, I'm referring to the fact that the context is that you're in the spec folder.

Keeping spec out of this makes refactoring down the line potentially easier as well. If this does become helpful to acceptance testing later, it'll be less change to move it if we don't include spec in the title. But perhaps that's too hypothetical to be used as evidence at this point.

Perhaps an alternative would be to get rid of `Test` as well, so it would just become `BeakerHelpers`. Does that sound better to you?